cloudevents/working-drafts/xml-format.md

```diff
@@ -135,19 +135,35 @@ Example:
 
 ### 3.2 Text Data
 
-Text MUST be carried in an element with a defined type of `xs:string`.
-The element MUST NOT contain any child elements.
+Text MUST be carried in an element with a defined type of `xs:string`. The content
```

Comment:
Minor thing, the doc seems to switch between "xs" and "xsi" - we mean the same thing in both case right? Should we pick one for consistency?